optaplanner-wb-ui/.gitignore

```diff
@@ -0,0 +1,12 @@
+/target
+/local
+/src/main/gwt-unitCache/
```

Comment:
All generated classes must go into the /target directory. Configure the maven plugins/IDE accordingly. There should be no reason to exclude classes in the /src dir

Reply:
Good spot! This is no longer necessary.